index.js

```diff
@@ -128,7 +128,11 @@ function eatFood() {
     updateFoodCell("remove", gameState.food);
     createFood();
     updateFoodCell("add", gameState.food);
-    gameState.score++;
+    if(gameState.score/5 > 2){
+      gameState.score += Math.floor(gameState.score/10);
```

Comment:
Great work! This works differently when score is higher than 20. I would suggest to use `Math.ceil()` function and give player 2 points if he have more than 10 points, 3 points if more than 20 and so on. Also in that case, you wouldn't need `if` statement.

Reply:
starting score is 0. floor and ceil both return 0. we can initialize score with 1 and remove if or use if incase score is 0.

Reply:
You can add 1 to score before dividing it by 10 and then you won't need if statement.